test/steps/inputs/1_given.js

```diff
@@ -23,6 +23,18 @@ import {
 	hasAncestorWithTitleMatching,
 } from '../utils';
 
+export function theUnvoteCanBeRetrievedFromItsSource() {
+	const { unvotePublicKeys, options: { unvote } } = this.test.ctx;
+	const publicKeysFromFile = unvotePublicKeys.map(votes => votes.trim()).join('\n');
```

Comment:
I meant, what if the file has e.g. new lines in? This `trim` doesn't do anything useful.